client/my-sites/people/viewers-list/viewers.jsx

```diff
@@ -157,4 +156,15 @@ class Viewers extends React.PureComponent {
 	}
 }
 
-export default connect( null, { recordGoogleEvent } )( localize( Viewers ) );
+const mapStateToProps = ( state, ownProps ) => ( {
+	viewers: getViewers( state, ownProps.site.ID ),
+	fetching: isFetchingViewers( state, ownProps.site.ID ),
+	totalViewers: getTotalViewers( state, ownProps.site.ID ),
+} );
```

Comment:
Since we only need the `site` prop we might as well simplify this to:

```suggestion
const mapStateToProps = ( state, { site } ) => ( {
	viewers: getViewers( state, site.ID ),
	fetching: isFetchingViewers( state, site.ID ),
	totalViewers: getTotalViewers( state, site.ID ),
} );
```

Reply:
done via 974dbea44b